lib/asset_sync/storage.rb

```diff
@@ -317,6 +342,7 @@ def upload_files
       end
 
       update_remote_file_list_cache(local_files_to_upload)
+      update_remote_file_list_in_remote if remote_file_list_remote_path
```

Comment:
Why not also put `if remote_file_list_remote_path` into `update_remote_file_list_in_remote`?

Reply:
Updated